migration/migrator/migrations/master/20230605230211_add_display_name_order_column.py

```diff
@@ -0,0 +1,178 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    # add column
+    database.execute("ALTER TABLE users ADD COLUMN IF NOT EXISTS display_name_order character varying(255) NOT NULL DEFAULT 'GIVEN_F';")
```

Comment:
It would be good to execute this entire migration in a single transaction via a single `database.execute()` call.  That way, any SQL error will result in the entire transaction getting rolled back if the migration fails.